packages/bits/demo/src/components/demo/wizard-v2/wizard-custom-layout/wizard-custom.component.html

```diff
@@ -16,7 +16,7 @@
                         [attr.aria-label]="step.ariaLabel || null"
                         [attr.aria-labelledby]="(!step.ariaLabel && step.ariaLabelledby) ? step.ariaLabelledby : null"
                         [index]="i"
-                        [state]="_getIndicatorType(i, step.state)"
+                        [state]="getStepState(i, step.state)"
```

Comment:
It's not your fault, @dmytrognatyshyn, but the function calls in this template seem pretty mysterious. It looks like getStepState is defined in the wizard directive. Usually, I would expect a method invoked in a template to be a member of this component instead of some other entity. fyi, @dmytrofirsov

Reply:
ah, I understand now. this is an extension of the NuiWizardDirective